doc/contributing.rst

```diff
@@ -121,8 +121,8 @@ follows the `NumPy docstring standard`_.
 
 Documentation is automatically built remotely during pull requests. If
 you want to also test documentation locally, you will need to install
-``sphinx sphinx-gallery sphinx_bootstrap_theme numpydoc``, and then within
-the ``mne/doc`` directory do:
+``sphinx sphinx-gallery sphinx_bootstrap_theme sphinx_fontawesome``, and then
+within the ``mne/doc`` directory do:
```

Comment:
we do need also numpydoc.

Reply:
it's in `environment.yml`, no need to have it here (I'm only listing things not in the env)